_config.yml

```diff
@@ -20,23 +20,23 @@
 # installation. Remember to keep the leading slash on the
 # `snippets` variable below!
 
-snippets: "/snippets_library/ComputeCanada_Graham_slurm"
```

Comment:
Looks like the top-level `config.yml` has been overwritten. Please revert. Local config should go in the `_config_options.yml` file in the folder for your site's snippets.

Reply:
Resolved by 76483ab.